### DISCUSSION TITLE
how can shipit revert a merged PR change?

### DISCUSSION THREAD

**Discussion by cgmcnew** [Shopify/shipit-engine#1201] [Q&A]
We are looking to understand how shipit can revert a merged PR change(s) after shipit detects that github has a failed condition on that merge.
We have the statuses from github in the shipit application for each merge and we would like to know if the functionality is built in to shipit already or not.
Thank you!

**Comment by casperisfine** [comment]
Shipit doesn't do reverts. Sorry.
